src/wolfgame.py

```diff
@@ -5586,35 +5586,51 @@ def role_stats(var, wrapper, rest):
 
     if wrapper.target != users.Bot:
         var.LAST_RSTATS = datetime.now()
-    
+
     if var.PHASE not in ("none", "join") and wrapper.target is not channel.Main:
             wrapper.pm(messages["stats_wait_for_game_end"])
             return
 
-    rest = rest.split()
-    if len(rest) == 0:
+    params = rest.split()
+    
+    if len(params) == 0:
         # this is a long message
         wrapper.pm(db.get_role_totals())
-    elif len(rest) == 1 or (rest[-1] == "all" and rest.pop()):
-        role = complete_role(var, wrapper, " ".join(rest))
-        if role:
-            wrapper.reply(db.get_role_stats(role))
-    else:
-        role = complete_role(var, wrapper, " ".join(rest[:-1]))
-        if not role:
+        return
+
+    role = complete_role(var, rest)
```

Comment:
Naming nit: Since there may be multiple roles returned, name the variable `roles` instead. This helps reduce confusion for other people looking at the code. Are we doing something with the first letter of the role?